### DISCUSSION TITLE
Can I activate a Conda environment in the docker when Flyte triggers a workflow

### DISCUSSION THREAD

**Discussion by drleowang** [flyteorg/flyte#3890] [Q&A] [answered]
My docker image used to register flyte workflows has a few Conda environments, can I activate one environment when a workflow is running in the docker?

Thanks!

**Comment by Future-Outlier** [comment]
I think the answer is yes!
You can use python sys module to do it!

**Comment by Future-Outlier** [comment, marked as answer]
@drleowang You can refer here to execute `conda` by shell task.
https://docs.flyte.org/projects/cookbook/en/v0.3.66/auto/core/flyte_basics/shell_task.html

**Comment by drleowang** [comment]
Thank you! It is a good practice!
